cmd/system-probe/modules/oom_kill_probe.go

```diff
@@ -44,7 +44,7 @@ type oomKillModule struct {
 }
 
 func (o *oomKillModule) Register(httpMux *module.Router) error {
-	httpMux.HandleFunc("/check/oom_kill", utils.WithConcurrencyLimit(utils.DefaultMaxConcurrentRequests, func(w http.ResponseWriter, req *http.Request) {
+	httpMux.HandleFunc("/oom_kill", utils.WithConcurrencyLimit(utils.DefaultMaxConcurrentRequests, func(w http.ResponseWriter, req *http.Request) {
```

Comment:
It looks like you reverted this to `/check/oom_kill`. I think it would be better to have this string be only `"/check"`. It would be namespaced by the module name, and end up being `/oom_kill_probe/check` in total.

You could modify `GetCheck(check string)` to request `http://unix/%s/check` and change the argument to be `oom_kill_probe` from `oom_kill`:
https://github.com/DataDog/datadog-agent/blob/74176b8f59689afe91ba29d34cb8dd0e2445b902/pkg/process/net/check.go#L26

WDYT?